infer/tests/create_test.go

```diff
@@ -88,6 +88,41 @@ func TestCreate(t *testing.T) {
 		}, resp.Properties)
 	})
 
+	t.Run("unwired-secrets", func(t *testing.T) {
```

Comment:
Should we test the condition where the input has the same name, but a different value in the output still creates a secret?

Reply:
We probably should. I'll add that test.

Reply:
Added in https://github.com/pulumi/pulumi-go-provider/pull/107/commits/4907e778f086ddaabc7d310e88969e52dd59b02d.